website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -53,6 +53,12 @@ Client agents will be configured to operate within a specific admin partition. T
 
 Values specified for [`proxy-defaults`](docs/connect/config-entries/proxy-defaults) configurations are scoped to a specific partition. Services registered in the partition will use the partition's `proxy-defaults` values.
 
+### Cross-partition Networking
+
+You can configure services to be discoverable and accessible by upstream services in any partition within the datacenter. Specify upstream services contained in other partitions by configuring the `partition-exports` configuration entry for the agent in the source partition. Refer to the [`partition-exports` documentation](docs/connect/config-entries/partition-exports) for details.
```

Comment:
A couple points here:
1. The services discovering an export are the downstreams
2. The second and third sentences kind of make it seem like the partition-exports are defined in the dialing partition
```suggestion
You can configure services to be discoverable and accessible by downstream services in any partition within the datacenter. Specify upstream services available for discovery by configuring the `partition-exports` configuration entry in the partition where they are registered. Refer to the [`partition-exports` documentation](docs/connect/config-entries/partition-exports) for details.
```